specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/examples/ApiManagementCreateDiagnostic.json

```diff
@@ -0,0 +1,118 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2018-06-01-preview",
+    "subscriptionId": "subid",
+    "diagnosticId": "applicationinsights",
+    "parameters": {
+      "properties": {
+        "enabled": true
```

Comment:
>true [](start = 19, length = 4)

CI is complaining about this property 
[https://travis-ci.org/Azure/azure-rest-api-specs/jobs/378880669](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/378880669)

Reply:
fixed